concepticondata/concepticon.tsv

```diff
@@ -3783,3 +3783,8 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3797	OSTRICH	Animals	Any of two species of large, flightless birds in the family Struthionidae, native to Africa. 	Person/Thing	
 3798	STINGER	The body	The pointed stinging organ of an animal, usually an insect, which is able to penetrate skin. Sometimes used to inject poison.	Person/Thing	
 3799	FISHBONE	The body	Any of the bones forming the skeleton of a fish, especially one of the sharp slender bones attached to the backbone. 	Person/Thing	
+3800	IVORY	The physical world	The hard, white substance which forms the tusks of animals like elephants, narwhals, or walruses and is often considered a valuable material.	Person/Thing	
+3801	DATE (FRUIT)	Agriculture and vegetation	The sweet, edible fruit of the date palm Phoenix dactylifera of the family Arecaceae.	Person/Thing	
+3802	FERN	Agriculture and vegetation	Any member of a large group of vascular plants in the family Filices who have neither flowers nor seeds.	Person/Thing	
+3803	RASPBERRY	Agriculture and vegetation	The edible, berry-like fruit of a multitude of plant species in the genus Rubus of the family Rosaceae, most typically red, small and sweet.	Person/Thing	
+3804	GRAPEFRUIT OR POMELO	Agriculture and vegetation	The fruit of the pomelo tree, Citrus maxima, of the family Rutaceae, or the fruit of the grapefruit tree, Citrus x paradisi, a citrus hybrid which originated from crossing pomelos and oranges. Due to the recent origin of the grapefruit and the close genetic relatedness between both fruits, they are often not distinguished.	Person/Thing	
```

Comment:
I would change the end of the last sentence:
", they are often not differentiated lexically."
Just to make it clear that they are given the same name.